example/styles.css

```diff
@@ -29,4 +29,20 @@ z-index: 10; }
   padding-bottom: 25px;
   margin-right: 65px;
   display: inline;
+}
+
+.badge {
+  width: 30px;
+  height: 30px;
+  padding: 4px 2px 0;
+  border-radius: 50px;
+  border: 2px solid #908f8f;
+  background: white;
+  color: black;
+  font-weight: bold;
+  font-size: 1.4em;
+  margin-left: 80%;
```

Comment:
Can you test if this works on different screen sizes and share screenshot also :)

Reply:
Totally slipped my mind. Will make sure to test it and share the screenshots.